webapp/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Could we use the same eslint config as the app.zetkin.org repo?

Reply:
I did that at the start but it doesn't work, something about prettier, it fail to read from prettier config....etc
so i just did the simple one
maybe if future we should make one repo for common config and lint that all repos imports

Reply:
Now it has been included, how ever few eslint disable has been added to be fixed later